sensu/plugins/check-apt-cache.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+# This checks that the apt cache has been updated recently.
+# It relies on the periodic cron job that touches FILE after a successful run.
+
+# the file that is touched by cron job on successful apt cache update
+FILE=/var/lib/apt/update-cache-success
+
+# arguments passed in for the warning/critical threshold in hours
+WARNING_HOURS=$1
+CRITICAL_HOURS=$2
+
+if [ $# -ne 2 ]; then
+    echo "usage: $0 WARNINGHOURS CRITICALHOURS"
+    exit 3
+fi
+
+if ! [[ -f $FILE ]]; then
+    echo "status file missing: $FILE"
```

Comment:
would it make sense have this part be idempotent, e.g. just create the file if it isn't there ?

Reply:
no, because this file is created/touched by the cron job that runs `apt-get update` on success

Reply:
oh I see, `update-cache-success` is built in sentinel for apt ?

Reply:
yes, I lay that down in ursula here https://github.com/blueboxgroup/ursula/pull/2993/files#diff-e39d5f40d5081a492e843ede007c8742R15